client/state/comments/selectors.js

```diff
@@ -56,12 +56,13 @@ export const getCommentById = createSelector(
 		);
 		return find( commentsForSite, comment => commentId === comment.ID );
 	},
-	( { state } ) => [
-		state.comments.items,
-		get( state.comments, 'items' ),
-		get( state.comments, 'errors' ),
-	]
+	( { state } ) => [ state.comments.items, state.comments.errors ]
 );
+
+export const getCommentErrors = state => {
```

Comment:
Should we put this with the newer selectors in `client/state/selectors`?

Reply:
can we defer this for a future pr?
i have Opinions on potentially merging this reducer with the `items` reducer